src/test/java/uk/gov/companieshouse/extensions/api/attachments/file/FileTransferApiClientUnitTest.java

```diff
@@ -112,10 +107,9 @@ public void testUpload_GenericExceptionResponse() {
 
         when(restTemplate.postForEntity(eq(DUMMY_URL), any(), eq(FileTransferApiResponse.class))).thenThrow(exception);
 
-        expectedException.expect(RestClientException.class);
-        expectedException.expectMessage(exception.getMessage());
-
-        fileTransferApiClient.upload(file);
+        RestClientException thrown = assertThrows(RestClientException.class, () ->
+            fileTransferApiClient.upload(file));
+        assertTrue(thrown.getMessage().contains(thrown.getMessage()));
```

Comment:
assertTrue(thrown.getMessage().contains(thrown.getMessage()));
should be
assertEquals(EXCEPTION_MESSAGE, thrown.getMessage());

Reply:
actually contains might be better.
assertTrue(thrown.getMessage().contains(EXCEPTION_MESSAGE));

Reply:
No the first one is better as it will return a more deailed error message should the test fail.